### PR TITLE
Add Unit Tests to check obfuscation of PII Data

### PR DESCRIPTION
**Pull request by aniruddhadave**
*Description of changes:*
- Add Unit Tests to check obfuscation of PII Data

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/csharp/PortingAssistant/PortingAssistant.UnitTests/PIIObfuscationUnitTest.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using System;
+using PortingAssistant.Client.Model;
+using System.Collections.Generic;
+
+namespace PortingAssistant.UnitTests
+{
+    [TestClass]
+    public class PIIObfuscationUnitTest
+    {
+        [TestMethod]
```
Comment (flywanli) [inline packages/csharp/PortingAssistant/PortingAssistant.UnitTests/PIIObfuscationUnitTest.cs:13]: could you use nunit to write unit test

File: packages/csharp/PortingAssistant/PortingAssistant.UnitTests/PIIObfuscationUnitTest.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+namespace PortingAssistant.UnitTests
+{
+    [TestFixture]
+    public class PIIObfuscationUnitTest
+    {
+        [Test]
+        public void TestSolutionPath() 
+        {
+            var solutionPath = "C:/Users/CustomerName/nopCommerce/src/NopCommerce.sln";
+            var encryptedSolutionPath = Crypto.SHA256(solutionPath); // 462eb7f46af82bd5155ef9f28ca3f5f638f702a7423b105478fa3d9267a344da
```
Comment (kanikaul-amazon) [inline packages/csharp/PortingAssistant/PortingAssistant.UnitTests/PIIObfuscationUnitTest.cs:17]: Do we need this comment?

File: packages/csharp/PortingAssistant/PortingAssistant.Common/Utils/TelemetryCollectionUtils.cs [hunk 3]
```diff
@@ -16,27 +16,60 @@ public static void CollectSolutionMetrics(Task<SolutionAnalysisResult> solutionA
...
+            TelemetryCollector.Collect<SolutionMetrics>(solutionMetrics);
+        }
+
+        public static void CollectProjectMetrics(ProjectAnalysisResult projectAnalysisResult, AnalyzeSolutionRequest request, string tgtFramework)
+        {
+            var projectMetrics = createProjectMetric(request.runId, request.triggerType, tgtFramework, projectAnalysisResult);
+            TelemetryCollector.Collect<ProjectMetrics>(projectMetrics);
+        }
+
+        public static void CollectNugetMetrics(Task<PackageAnalysisResult> result, AnalyzeSolutionRequest request, string tgtFramework)
```
Comment (kanikaul-amazon) [inline packages/csharp/PortingAssistant/PortingAssistant.Common/Utils/TelemetryCollectionUtils.cs:29]: nit: rename to packageAnalysisResult for consistency

**Review by kanikaul-amazon — CHANGES_REQUESTED** [review]
Hey thanks for adding this. Left a few minor comments

**Review by kanikaul-amazon — APPROVED** [review]

**Pull request merged**